packages/mosip-api/src/mosip-api.ts

```diff
@@ -194,7 +194,7 @@ export const postBirthRecord = async ({
 
   const requestBody = JSON.stringify(
     {
-      id: composition.id,
```

Comment:
MOSIP informed that the record/composition UUID probably doesn't work here, but my pull request on SQLite should address this issue.